xapian-core/configure.ac

```diff
@@ -801,6 +801,7 @@ using namespace std;
     [ac_cv_func_snprintf=unknown;break]
   )
 done
+
```

Comment:
Please avoid random whitespace changes in patches - they make review a little harder, and add noise for people trying to trace back through the history of the code.

Reply:
I pushed hurriedly, I apologize. Fixed.